### PR TITLE
Scheduling Tutorial

### PR DESCRIPTION
**Pull request by kvyh**
I have added  a scheduling tutorial to the docs. It relies on a couple of the changes from #203, though isn't branched from there, so it will fail tests (maybe?) until that is merged.

### CODE REVIEW

**Comment by bmorris3** [comment]
This will eventually close #200.

Would you also remove the notebooks in this PR?

File: docs/tutorials/scheduling.rst [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+Say we want to observe Vega, Arcturus, Deneb, and Polaris, as well as a few
+other stars in Ursa Minor. For this we will be using the Apache Point Observatory.
+
+First we define our `~astroplan.Observer` object (where we are observing from):
+
+.. code-block:: python
+
+    >>> from astroplan import Observer
+
+    >>> apo = Observer.at_site('apo)
```
Comment (bmorris3) [inline docs/tutorials/scheduling.rst]: That's going to need another `'`.

Comment (bmorris3) [inline docs/tutorials/scheduling.rst]: Don't forget to build locally with both Python 2 and 3 before pushing to GitHub to catch things like this, with 
```
python setup.py build_sphinx -lo
```

**Comment by kvyh** [comment]
The only sphinx warning remaining is one that should be fixed by #203. The code examples are not good for doctests (line breaks apparently cause trouble) so they are now skipped.

**Comment by kvyh** [comment]
This is superseded by #218 so I will close it.

**Pull request closed without merging**